test/e2e/config_test.go

```diff
@@ -1078,3 +1012,26 @@ func assertRemoteWriteWasSet(namespace, crName, urlValue string) func(t *testing
 		}
 	}
 }
+
+func assertEnforcedTargetLimit(limit uint64) func(*testing.T) {
```

Comment:
this should be moved to `common_assertions`

Reply:
this comment still applies

Reply:
I would have to strongly disagree with moving this test, because it is not beneficially to expose it on the framework directly since it specific as not only a config test but also the uwm.

I feel the contents of the test alone suggest that it is totally non-reusable in any other context.

If we move this out and continue to move out every non-generic helper function (a test) then the framework becomes littered with tests and makes it more confusing. The idea with moving out those assertions is to clean up the tests as a whole with these frequently used functions which are generally pre-reqs to doing more specific tests such as `assertEnforcedTargetLimit`

Reply:
SG